index.html

```diff
@@ -1,192 +1,192 @@
-                <li><a href="https://angel.co/u/dany-augustin"><img src="img/angellist.svg" alt="angellist-icon"></a></li>
-                <li><a href="www.twitter.com/amiralda"><img src="img/twitter.svg" alt="twitter-icon"></a></li>
-                <li><a href="https://www.medium.com/amiralda/"><img src="img/medium.svg" alt="medium-icon"></a></li>
-            </ul>
-            <a href="#" class="preview">Get my resume</a>
-        </div>
-        <ul class="skills">
-            <div>
-                <li class="languages">
-                    <h3 class="language-title">Languages</h3>
-                    <i class="arrow down"></i>
             </div>
-            <ul class="language-description">
-                <li class="skills-parent">
-                    <img class="language-icon" src="img/js.png" alt="javascript">
-                    <p class="name">Javascript</p>
-                </li>
-                <li class="skills-parent">
-                    <img class="language-icon" src="img/html.png" alt="html">
-                    <p class="name">HTML</p>
-                </li>
-                <li class="skills-parent">
-                    <img class="language-icon" src="img/css.png" alt="css">
-                    <p class="name">CSS</p>
-                </li>
-            </ul>
-            </li>
-            <li class="languages separator">
-                <h3 class="language-title">Frameworks</h3>
-                <i class="arrow right"></i>
-            </li>
-            <li class="languages separator">
-                <h3 class="language-title">Skills</h3>
-                <i class="arrow right"></i>
-            </li>
-            <li class="languages separator">
-                <h3 class="language-title"></h3>
-            </li>
-        </ul>
-
         
-    </section>
-    </section>
-    <section class="contact">
-        <h2 class="title-contact">Contact me</h2>
-        <img class="text-image">
-        <p class="text-contact">If you have an application you are interested in developing, a feature that you need built or a
-            project that needs coding. I’d love to help with it </p>
-        <form action="https://formspree.io/f/mwkjbodb" method="POST">
-            <input type="text" name="name" id="name" placeholder="Name" maxlength="30" required>
-            <input type="email" name="email" id="email" placeholder="Email" required>
-            <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here" maxlength="500"
-                required></textarea>
-            <button type="submit"id="contact-button">Get in touch</button>
-           <p class="separator"></p>
-        </form>
-       
-    </section>
-    
-</body>
+        </section>
+        </section>
+        <section class="contact">
+            <h2 class="title-contact">Contact me</h2>
+            <!-- <img class="text-image"> -->
```

Comment:
- [ ] Well done with this milestone :muscle:  This seem to be line of code no longer in use. It is good practice to always keep your code clean by removing comments that are not significantly important to aid clarity. :writing_hand: